--- menu.php	2012-10-14 17:20:46.000000000 +0530
+++ menu.php1	2012-10-14 17:18:33.000000000 +0530
@@ -65,7 +65,7 @@
 $menu_items['section_configuration'] = _('Configuration');
 $menu_items['view_config'] = _('View config');
 $menu_items['my_account'] = _('My Account');
-$menu_items['backup_restore'] = _('Backup/Restore');
+//$menu_items['backup_restore'] = _('Backup/Restore');
 
 # menu structure using array keys from translated labels above
 $menu = array(
@@ -77,7 +77,8 @@
 		'comments', 'schedule_downtime', 'process_info', 'scheduling_queue', 'performance_info', 'hyper_map'), //'nagvis'), /* remove hardcoded nagvis menu entry */
 	'section_reporting' => array('trends', 'pnp', 'alert_history', 'alert_summary', 'notifications', 'event_log',
 		'availability', 'sla', 'schedule_reports', 'statistics'),
-	'section_configuration' => array('view_config', 'my_account', 'backup_restore', 'configure')
+	//'section_configuration' => array('view_config', 'my_account', 'backup_restore', 'configure')
+	'section_configuration' => array('view_config', 'my_account', 'configure')
 );
 
 $all_host_status_types = nagstat::HOST_PENDING|nagstat::HOST_UP|nagstat::HOST_DOWN|nagstat::HOST_UNREACHABLE;
@@ -135,7 +136,7 @@
 	$menu_items['section_configuration'] => array(
 		$menu_items['view_config'] 				=> array('/config', 'viewconfig',0),
 		$menu_items['my_account'] 				=> array('/user', 'password',0),
-		$menu_items['backup_restore']			=> array('/backup', 'backup',0)
+//		$menu_items['backup_restore']			=> array('/backup', 'backup',0)
 	)
 );
 
